svn/common.py

```diff
@@ -157,8 +157,11 @@ def properties(self, rel_path=None):
         # query the proper list of this path
         root = xml.etree.ElementTree.fromstring(result)
         target_elem = root.find('target')
-        property_names = [p.attrib["name"]
-                          for p in target_elem.findall('property')]
+        if target_elem is not None:
+            property_names = [p.attrib["name"]
+                              for p in target_elem.findall('property')]
+        else:
```

Comment:
When is this the case?

Reply:
Any time there are no properties associated, the "target" element it's searching for won't be found. 
Examples:
```
http://svn.apache.org/repos/asf/activemq/activecluster/trunk/src/main/java/org/apache/activecluster@1808406
<?xml version="1.0" encoding="UTF-8"?>
<properties>
</properties>

http://svn.apache.org/repos/asf/activemq/activecluster/trunk/src/main/java/org/apache/activecluster/Cluster.java@1808406
<?xml version="1.0" encoding="UTF-8"?>
<properties>
  <target path="http://svn.apache.org/repos/asf/activemq/activecluster/trunk/src/main/java/org/apache/activecluster/Cluster.java">
    <property name="svn:eol-style"/>
  </target>
</properties>
```

Reply:
Oh, I got it. Thanks for clarifying.